meerk40t/core/elements.py

```diff
@@ -1617,14 +1707,52 @@ def op_disable(command, channel, _, data=None, **kwrgs):
         )
         def op_enable(command, channel, _, data=None, **kwrgs):
             for op in data:
-                op.output = True
-                channel(_("Operation '%s' enabled.") % str(op))
-                op.notify_update()
+                no_op = True
+                if hasattr(op, "output"):
+                    try:
+                        op.output = True
+                        channel(_("Operation '%s' enabled.") % str(op))
+                        op.notify_update()
+                        no_op = False
+                    except AttributeError:
+                        pass
+                if no_op:
+                    channel(_("Operation '%s' can't be enabled.") % str(op))
             return "ops", data
 
         # ==========
         # ELEMENT/OPERATION SUBCOMMANDS
         # ==========
+        @self.console_command(
+            "lock",
+            help=_("Lock element (protect from manipulation)"),
+            input_type=("elements"),
+            output_type=("elements"),
+        )
+        def e_lock(data=None, **kwargs):
```

Comment:
This is probably a good idea and should be implemented reasonably. While regmark has it's place that place is mostly in actually providing marks for position or a jig or whatnot. There's some need to keep elements ready to burn but prevent them from being selected or messed with.